### PR TITLE
Add authentication

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review. Added Auth. Yet to add Private routing (please ignore that for now)

https://ecomm-app-zeta.vercel.app/

### CODE REVIEW

File: src/context/auth-context.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const [loggedIn, setLoggedIn] = useState(false);
+  const userToken = localStorage.getItem("userToken");
+
+  useEffect(() => {
+    if (userToken) {
+      setLoggedIn(true);
+    }
+  }, [userToken]);
+
+  const logoutHandler = () => {
```
Comment (deekshithmd) [inline src/context/auth-context.js:18]: suggestion: you can write this handler in the page where you are created logout button.

**Review by deekshithmd — APPROVED** [review]

File: src/Pages/SignupPage/SignupPage.css [hunk 2]
```diff
@@ -60,7 +62,7 @@
 }
```
Comment (ashishkatiyar2404) [inline src/Pages/SignupPage/SignupPage.css:62]: remove class if you are not styling it.

**Review by ashishkatiyar2404 — APPROVED** [review]
The login and signup page looks good to me.
You can merge now.

Suggestion: - pls deploy ur branch so the reviewer can see your UI also or you can add screenshots also.

File: src/context/auth-context.js (hunk 1 above)
Comment (ShrutiLakhani) [reply src/context/auth-context.js:18]: Done

**Pull request merged**